rep-2004.rst

```diff
@@ -636,43 +636,43 @@ Quality Declaration Template
 
   Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level N in REP-2004](https://www.ros.org/reps/rep-2004.html).
 
-  ## Version Policy
+  ## [1] Version Policy
 
-  ### Version Scheme
+  ### [1.i] Version Scheme
 
-  ### API Stability Within a Released ROS Distribution
+  ### [1.iii] Public API Declaration
 
-  ### ABI Stability Within a Released ROS Distribution
+  ### [1.iv]/[1.vi] API Stability Within a Released ROS Distribution
 
-  ### Public API Declaration
+  ### [1.v]/[1.vi] ABI Stability Within a Released ROS Distribution
 
-  ## Change Control Process
+  ## [2] Change Control Process
```

Comment:
Here too I think it's valuable to break it into all the items, so packages can explicitly mention each one of them. They can always combine them all into a single link if they want, for example. :thinking: